src/pybind/mgr/dashboard/controllers/cephfs.py

```diff
@@ -673,17 +673,21 @@ def ls_dir(self, fs_id, path=None, depth=1):
 @APIDoc('CephFS Subvolume Management API', 'CephFSSubvolume')
 class CephFSSubvolume(RESTController):
 
-    def get(self, vol_name: str):
+    def get(self, vol_name: str, group_name: str = ''):
```

Comment:
```suggestion
    def get(self, vol_name: str, group_name: str = None):
```

Reply:
Left is as `group_name: str = ""` since it is a string